packages/pwashortcut.rb

```diff
@@ -0,0 +1,181 @@
+require 'package'
+
+class Pwashortcut < Package
+  description 'Scripts to create a PWA shortcut on ChromeOS dock for Chromebrew GUI Apps.'
+  homepage 'https://github.com/supechicken666/pwashortcut/'
+  version '1.3.1'
+  source_url 'https://github.com/supechicken666/pwashortcut/archive/1.3.1-debug.tar.gz'
+  source_sha256 'ec4d3e728733550a8cdaa8b8f19c5df01982f9e39dfc1a456e39d52934bf9d89'
+
+  HOME = Dir.home
```

Comment:
Where is this variable used?

Reply:
Getting the `~` directory path for `FileUtils`